narwhal/executor/src/lib.rs

```diff
@@ -117,42 +96,32 @@ pub async fn get_restored_consensus_output<State: ExecutionState>(
     certificate_store: CertificateStore,
     execution_state: &State,
 ) -> Result<Vec<CommittedSubDag>, SubscriberError> {
-    // We always want to recover at least the last committed certificate since we can't know
+    // We always want to recover at least the last committed sub-dag since we can't know
     // whether the execution has been interrupted and there are still batches/transactions
-    // that need to be send for execution.
+    // that need to be sent for execution.
 
-    let last_committed_leader = execution_state
-        .load_execution_indices()
-        .await
-        .last_committed_round;
+    let last_executed_sub_dag_index = execution_state.last_executed_sub_dag_index().await;
 
     let compressed_sub_dags =
-        consensus_store.read_committed_sub_dags_from(&last_committed_leader)?;
```

Comment:
Perhaps we can rename `compressed_sub_dag(s)` -> `sub_dag(s)` here and in the loop below, since we no longer compress subdugs. (Which btw I liked, since it simplified consensus recovery and it is much easier to reason about it right now)